src/20_Components/amp-carousel.html

```diff
@@ -90,7 +96,29 @@
       This is a green box.
     </div>
   </amp-carousel>
-  <!-- A good use case for `amp-carousel` are image galleries, [here](/advanced/image_galleries_with_amp-carousel/) are some examples. -->
+
+  <!-- You can also add other AMP components besides <amp-img> to the carousel.  -->
+  <amp-carousel height="300" layout="fixed-height" type="slides">
+    <amp-soundcloud height=300
+        layout="fixed-height"
+        data-trackid="243169232">
+    </amp-soundcloud>
+    <amp-youtube
+        data-videoid="mGENRKrdoGY"
+        width="400" height="300">
```

Comment:
Can you use a fill layout to avoid duplicating width and height?

Reply:
Done